### ISSUE TITLE
Can't generate key with version 1.0.9-15.1

### ISSUE THREAD

**Issue by atlochowski** [monich/harbour-foilpics#23]
When I type password and tap on generate button nothing happening. 
Here you have output of journalctl:
```
ov 09 11:39:11 Sailfish systemd[1225]: Starting Transfer engine...
Nov 09 11:39:12 Sailfish dbus-daemon[1302]: dbus-daemon[1302]: [session uid=100000 pid=1302] Successfully activated service 'org.nemo.transferengine'
Nov 09 11:39:12 Sailfish systemd[1225]: Started Transfer engine.
Nov 09 11:39:13 Sailfish invoker[17895]: WARNING: An inactive plugin is misbehaving - tried to show a window!
Nov 09 11:39:13 Sailfish invoker[17895]: WARNING: requestActivate() called for  QQuickView(0x1637808)  which has Qt::WindowDoesNotAcceptFocus set.
Nov 09 11:39:13 Sailfish lipstick[17854]: [W] unknown:0 - WindowPixmapItem does not have a source texture, cover will be dropped..
Nov 09 11:39:13 Sailfish booster-silica-qt5[20828]: [D] unknown:0 - Using Wayland-EGL
Nov 09 11:39:13 Sailfish mapplauncherd[20828]: SilicaBooster: Initiate asynchronous preload.
Nov 09 11:39:18 Sailfish invoker[17895]: WARNING: requestActivate() called for  QQuickView(0x1637808)  which has Qt::WindowDoesNotAcceptFocus set.
Nov 09 11:39:26 Sailfish [20282]: [W] unknown:17 - file:///usr/share/harbour-foilpics/qml/GenerateKeyView.qml:17: ReferenceError: passphrase is not defined
```

**Comment by monich** [comment]
ouch

**Comment by monich** [comment]
QML desperately needs a compiler! Which would at least check the syntax :)

**Comment by monich** [comment]
Should be fixed in 1.0.10. Sorry about the trouble and thanks for the report!
